examples/cosmos/chain_export_test.go

```diff
@@ -29,39 +28,15 @@ func CosmosChainStateExportTest(t *testing.T, name, version string) {
 	numVals := 1
 	numFullNodes := 0
 
-	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
-		{
-			Name:      name,
-			ChainName: name,
-			Version:   version,
-			ChainConfig: ibc.ChainConfig{
-				Denom: "ujuno",
-			},
-			NumValidators: &numVals,
-			NumFullNodes:  &numFullNodes,
-		},
-	})
-
-	chains, err := cf.Chains(t.Name())
-	require.NoError(t, err)
+	cfg := ibc.ChainConfig{
+		Denom: "ujuno",
```

Comment:
I realize you're just updating the test to use the new helper functions but I'm pretty sure we could pass in an empty ibc.ChainConfig{} and it will still work since it will use the "pre configured" juno.

https://github.com/strangelove-ventures/interchaintest/blob/08b20f60d3686c535a6828a7216e529f5bdede75/configuredChains.yaml#L462-L474

More of a note. You're call if you want to remove.